source/parser/comptime.civet

```diff
@@ -203,22 +194,22 @@ function serialize(value: ???): string
         stack.add val
         str :=
           switch Object.getPrototypeOf val
-            when RegExp::
+            when RegExp::, context?.RegExp?::
```

Comment:
Are both `?`s necessary? If `context` is `undefined` the rest of the chain is skipped.

Reply:
Fair point. I think in the future we might want to add *more* `?`s before `::`, for both local and remote checks, but given that we currently require that `RegExp` (say) is present, we can assume that it's also in `context`, given how we copy over all globals. (I wonder if there are JS runtimes e.g. with no `TypedArray` support. But we can wait for that to arise.)